.github/workflows/cypress-common.yml

```diff
@@ -0,0 +1,55 @@
+name: Cypress reusable workflow
+
+on:
+  workflow_call:
+    inputs:
+      scope:
+        description: 'scope name'
+        required: true
+        type: string
+      with-react-17:
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
[Note]

> **continue-on-error** applies to a single job. If **continue-on-error** is true, other jobs in the matrix will continue running even if the job with continue-on-error: true fails.

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures